beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/DepositProviderTest.java

```diff
@@ -305,6 +305,21 @@ void shouldLogAnEventOnSlotWhenAllDepositsRequiredForStateNotAvailable() {
     verify(eventLogger).eth1DepositDataNotAvailable(UInt64.valueOf(9), UInt64.valueOf(10));
   }
 
+  @Test
```

Comment:
I'd test also that getDeposits() is empty since former mechanism is disabled

Reply:
This was part of a previous PR, can have a look at `noDepositsIncludedIfFormerDepositMechanismHasBeenDisabled` and `getsRemainingEth1PendingDepositsIfElectraIsEnabled`